librz/analysis/rzil/rzil.c

```diff
@@ -0,0 +1,59 @@
+#include <rz_analysis.h>
+
+RZ_API RzAnalysisRzil *rz_analysis_rzil_new() {
+        RzAnalysisRzil *rzil = RZ_NEW0(RzAnalysisRzil);
+        if (!rzil) {
+                return NULL;
+        }
+        rzil->vm = RZ_NEW0(struct rz_il_vm_t);
+        if (!rzil->vm) {
+                free(rzil);
+                return NULL;
+        }
+        return rzil;
+}
+
+RZ_API void rz_analysis_rzil_cleanup(RzAnalysisRzil *rzil, RzAnalysis *analysis) {
```

Comment:
Check for `rzil`/`analysis` being NULL, also make `analysis` a first argument in those functions